internal/pkg/cli/pipeline_show.go

```diff
@@ -129,7 +145,7 @@ func (o *showPipelineOpts) askPipelineName() error {
 	}
 
 	if len(pipelineNames) == 0 {
-		log.Infof("No pipelines found for project %s\n.", color.HighlightUserInput(o.AppName()))
+		log.Infof("No pipelines found for application %s.\n", color.HighlightUserInput(o.AppName()))
```

Comment:
Super nit: when do we add a period at the end of a sentence for a single sentence? The line 138 above doesn't have one. Just want to make sure for my future PRs :)

Reply:
Hm, @efekarakus would be the one for this but I've seen both opinions.  [Here](https://dave.cheney.net/2015/11/05/lets-talk-about-logging) it looks like there isn't punctuation, so I can take that out. Good catch!